debian/conf/distributions

```diff
@@ -149,3 +149,19 @@ Architectures: amd64 i386 source
 Components: main prerelease
 SignWith: F473DD4473365DE1
 Tracking: minimal
+
```

Comment:
Shouldn't we remove `lunar` and `mantic`, to align with https://github.com/element-hq/synapse/pull/17824/files?

Reply:
We have kept all the other old ubuntu versions. I think probably so things don't break for people running them